### ISSUE TITLE
Largest Contentful Paint image link not updated, when switching between reruns of the page

### ISSUE THREAD

**Issue by JoshuvaGeorge03** [sitespeedio/sitespeed.io#4218]
### Have you read the documentation?

- [X] Yes, I've read the [how to make a reproducable bug guide](https://www.sitespeed.io/documentation/sitespeed.io/bug-report/)
- [X] Yes, I've read the [how to debug my script guide](https://www.sitespeed.io/documentation/sitespeed.io/scripting/#debug)

### URL

https://example.com 

### What are you trying to accomplish

KFYI: I have did not provided the url, because, this bug is so straightforward.

Thanks for this great tool and I am using sitespeed.io to measure webpage, and I am doing 2 runs for same pages...

I have looked at metrics section,  checked largest contetful paint images between 2 runs, On one run of the tool, LCP is not found, another run it was found, but, In LCP images, it was not reflected...

then I try to debug, why it was happening, and found out, it was happening because url of LCP images not reflected in regards to the number of runs of web page selected, please refer the screenshot for more info...

if you have any doubts, and need any further clarifications, happy to help

![sitespeed-bug-report](https://github.com/sitespeedio/sitespeed.io/assets/39693066/491b3f68-c828-4db2-ac6b-fb05b14e5f88)
 

### What browser did you use?

Chrome

### How to reproduce

```shell
Measure any webpage...

Make number of runs of webpage to 2, 

after reported generated successfully, go to the report, and to metrics section, check out lcp image url, ensure whether it is changing when switching between runs of web page url.
```

**Comment by soulgalore** [comment]
Hi @JoshuvaGeorge03 thanks for creating the issue. Let me have a look.

**Comment by JoshuvaGeorge03** [comment]
Thanks for the quick response

**Comment by soulgalore** [comment]
I fixed that now in that #4221 , there's a new release coming later today when I tested the new version of Firefox (release date today).

**Comment by JoshuvaGeorge03** [comment]
Thanks very much for the quick turn around...
